docs/release-notes.md

```diff
@@ -4,7 +4,19 @@ nav_order: 8
 
 # Release notes
 
-## Upcoming Afterburn 5.5.0 (unreleased)
+## Upcoming Afterburn 5.5.1 (unreleased)
```

Comment:
Releases are usually minor version bumps, so that's what we optimize for. So this would be 5.6.0.